sdk/src/compute_budget.rs

```diff
@@ -22,6 +22,7 @@ crate::declare_id!("ComputeBudget111111111111111111111111111111");
 )]
 pub enum ComputeBudgetInstruction {
     /// Deprecated
+    /// TODO: after feature remove_deprecated_request_unit_ix::id() is activated, replace it with 'unused'
```

Comment:
This shouldn't be a "doc" comment because it's an internal comment
```suggestion
    // TODO: after feature remove_deprecated_request_unit_ix::id() is activated, replace it with 'unused'
```

Reply:
This comment still needs to be updated